README.md

```diff
@@ -16,6 +16,8 @@ Then:
     cd firefox-ui-tests
     python setup.py develop
 
+If you do not have installed virtualenv and virtualenvwrapper you can also use the create_venv.py script to let it automatically create a virtual environmnet for you.
+
```

Comment:
this isn't clear on how to get into a virtualenv.  The script being added is great for downloading and installing virtualenv, but for actually creating  a virtualenv it doesn't look simple.  Could we extend this new script to use an existing one, create one if virtualenv exists, or do what it does if all else fails?